<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveEvaluation</class>
 <widget class="QDialog" name="SaveEvaluation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Task Grade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Task Feedback:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Evaluation Status:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createEvaluation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Evaluation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Evaluation for task #1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editEvaluation">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Evaluation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteEvaluation">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>160</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>DeleteEvaluation</string>
   </property>
  </widget>
  <widget class="QLabel" name="taskGrade">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="taskFeedback">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="evalStatus">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
